pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -112,3 +113,19 @@ def __verify_dp_batch_transfer_support(self, model: 'pl.LightningModule') -> Non
         for hook in batch_transfer_hooks:
             if self.trainer.accelerator_connector.use_dp and is_overridden(hook, model):
                 raise MisconfigurationException(f'Overriding `{hook}` is not supported in DP mode.')
+
+    def __verify_manual_optimization_support(self, model: 'pl.LightningModule') -> None:
+        if model.automatic_optimization:
+            return
+        if self.trainer.gradient_clip_val > 0:
+            raise MisconfigurationException(
+                f"Automatic gradient clipping is not supported for manual optimization."
+                f" Remove the Trainer setting Trainer(gradient_clip_val={self.trainer.gradient_clip_val})"
```

Comment:
nit: redundant wording

```suggestion
                f" Remove `Trainer(gradient_clip_val={self.trainer.gradient_clip_val})`"
```

Reply:
do you by any chance know why we add backticks to error and warning messages?

Reply:
Just to differentiate between code and text. Same pattern as for docstrings